superset/assets/src/SqlLab/components/QueryAutoRefresh.jsx

```diff
@@ -19,10 +19,12 @@ class QueryAutoRefresh extends React.PureComponent {
   shouldCheckForQueries() {
     // if there are started or running queries, this method should return true
     const { queries } = this.props;
-    const queryKeys = Object.keys(queries);
-    const queriesAsArray = queryKeys.map(key => queries[key]);
-    return queriesAsArray.some(q =>
-      ['running', 'started', 'pending', 'fetching'].indexOf(q.state) >= 0);
+    const now = new Date().getTime();
+    return Object.values(queries)
+      .some(
+        q => ['running', 'started', 'pending', 'fetching'].indexOf(q.state) >= 0 &&
+        now - q.startDttm < 21600000,
```

Comment:
No hard coded values please. Put it in `src/SqlLab/constants.js`

Reply:
Done